### PR TITLE
[PLAT-1197] Fix download counts on legacy preprints 

### PR DESCRIPTION
**Pull request by caseyrollins**
#### Purpose
Fix download counts on legacy preprints by updating `_id`s in the `PageCounter` table to use preprint guids instead of project guids. 

#### Changes
Changed existing migration to update `PageCounter` entries when legacy preprints are migrated (h/t @pattisdr)

#### Side Effects
None expected

#### Ticket
https://openscience.atlassian.net/browse/PLAT-1197

### CODE REVIEW

File: osf/migrations/0136_preprint_node_divorce.py [hunk 1]
```diff
@@ -51,6 +54,12 @@ def reverse_func(apps, schema_editor):
         node.preprint_file = preprint_file
         preprint.primary_file = None
 
+        node_id = Guid.objects.get(content_type=ContentType.objects.get_for_model(AbstractNode).id, object_id=node.id)._id
+        preprint_id = Guid.objects.get(content_type=ContentType.objects.get_for_model(Preprint).id, object_id=preprint.id)._id
+        PageCounter.objects.filter(Q(_id__contains=preprint_id) & Q(_id__contains=preprint_file._id)).update(
```
Comment (pattisdr) [inline osf/migrations/0136_preprint_node_divorce.py]: This will break here if preprint_file is None.

**Review by pattisdr — CHANGES_REQUESTED** [review]
Just the one change requested - I hope this doesn't slow down the reverse migration too much, but then, the reverse migration wasn't really optimized for speed.  

Thanks for your work @caseyrollins!

File: osf/migrations/0136_preprint_node_divorce.py (hunk 1 above)
Comment (caseyrollins) [reply osf/migrations/0136_preprint_node_divorce.py]: Good catch. Will move this block into the above `if` block.

**Review by pattisdr — APPROVED** [review]
Looks good to me!

**Pull request merged**